src/features/beacons/selectors.ts

```diff
@@ -10,7 +12,7 @@ import { selectOrdered } from '~/utils/collections';
  * on the UI.
  */
 export const getBeaconsInOrder = createSelector(
-  (state) => state.beacons,
+  (state: RootState) => state.beacons,
```

Comment:
Instead of annotating these sub-selectors inline, I would prefer creating a properly typed reusable selectors, such as:
```ts
export const getBeaconsAsCollection: AppSelector<Collection<Beacon>> = (
  state
) => state.beacons;
```